### PR TITLE
feat: add navbar + hero section; the navbar and the hero are responsive

### PR DESCRIPTION
**Pull request by mkubdev**
### Describe the changes you've made

Added a Tailwind Navbar and the Hero section to match the Figma Design. Navbar and Hero are responsive.

### CODE REVIEW

**Comment by shuklaritvik06** [comment]
Hey! Thanks for the contribution! Can you please share the screenshots?

**Comment by mkubdev** [comment]
Sure!
Capture

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :) Just one suggestion I want to give , when we hove on the explore now button , there is no need for the text decoration, maybe @vinzvinci  what you think?

**Comment by vinzvinci** [comment]
We can include that next time

**Comment by vinzvinci** [comment]
Also the logo in the navbar

**Comment by mkubdev** [comment]
@shuklaritvik06 yes, i'll remove it next commit

**Comment by shuklaritvik06** [comment]
> @shuklaritvik06 yes, i'll remove it next commit

Yeah Sure! Btw Great Work!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          if (t == elm) {
+            return true;
+          }
+          t = t.parentNode;
+        }
+        return false;
+      }
+    </script>
+  </body>
+</html>
```
Comment (Panquesito7) [inline index.html:143]: Missing an endline here.

Comment (mkubdev) [reply index.html:143]: yes, forget it...  added it in the latest commit https://github.com/web3community/web3community.github.io/commit/4caa75e002823833c9e0d1974c1ec52c44dd2515

**Pull request merged**